### PR TITLE
Changelog v47.10.0

### PR DESCRIPTION
**Pull request by AnanyaJha**
### What does this PR do?

Creates changelog for release v 47.10.0

### CODE REVIEW

File: packages/salesforcedx-vscode/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,17 @@
+# 47.10.0 - (INSERT RELEASE DATE [Month Day, Year])
```
Comment (brpowell) [inline packages/salesforcedx-vscode/CHANGELOG.md]:
```suggestion
# 47.10.0 - December 12, 2019
```

Comment (sfsholden) [inline packages/salesforcedx-vscode/CHANGELOG.md]: Date - this piece isn't automated yet

File: packages/salesforcedx-vscode/CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,17 @@
...
+
+#### docs
+
+- Fix typo in doc title ([PR #1830](https://github.com/forcedotcom/salesforcedx-vscode/pull/1830))
+
+## Added
+
+#### salesforcedx-vscode-core
+
+- Added Japanese translations for labels and messages ([PR #1551](https://github.com/forcedotcom/salesforcedx-vscode/pull/1551))
```
Comment (sfsholden) [inline packages/salesforcedx-vscode/CHANGELOG.md]: Maybe we should add a link to the docs that were translated? Not sure if it's needed though.

Comment (brpowell) [reply packages/salesforcedx-vscode/CHANGELOG.md]: I think they were just label changes in the extensions.

**Comment by brpowell** [comment]
Also need to add https://github.com/forcedotcom/salesforcedx-vscode/pull/1836 which fixes https://github.com/forcedotcom/salesforcedx-vscode/issues/1832

File: packages/salesforcedx-vscode/CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,21 @@
+# 47.10.0 - December 12, 2019
+
+## Fixed
+
+#### docs
+
+- Fix typo in doc title ([PR #1830](https://github.com/forcedotcom/salesforcedx-vscode/pull/1830))
```
Comment (brpowell) [inline packages/salesforcedx-vscode/CHANGELOG.md]:
```suggestion
- Fix typo in doc title for [Lightning Web Components](https://developer.salesforce.com/tools/vscode/en/lwc/writing) ([PR #1830](https://github.com/forcedotcom/salesforcedx-vscode/pull/1830))
```

Suggested by Bindu

File: packages/salesforcedx-vscode/CHANGELOG.md [hunk 4]
```diff
@@ -1,3 +1,21 @@
...
+
+## Fixed
+
+#### docs
+
+- Fix typo in doc title for [Lightning Web Components]( forcedotcom.github.io/salesforcedx-vscode/articles/lwc/writing)
+
+#### salesforcedx-vscode-lightning
+
+-  Fix indexing in Lightning Web Components  ([PR #1836](https://github.com/forcedotcom/salesforcedx-vscode/pull/1836), [Issue #1832](https://github.com/forcedotcom/salesforcedx-vscode/issues/1832))
```
Comment (sfsholden) [inline packages/salesforcedx-vscode/CHANGELOG.md]:
```suggestion
- Fix indexing in Lightning Web Components ([PR #1836](https://github.com/forcedotcom/salesforcedx-vscode/pull/1836), [Issue #1832](https://github.com/forcedotcom/salesforcedx-vscode/issues/1832))
```

File: packages/salesforcedx-vscode/CHANGELOG.md [hunk 5]
```diff
@@ -1,3 +1,21 @@
+# 47.10.0 - December 12, 2019
+
+## Fixed
+
+#### docs
+
+- Fix typo in doc title for [Lightning Web Components]( forcedotcom.github.io/salesforcedx-vscode/articles/lwc/writing)
```
Comment (sfsholden) [inline packages/salesforcedx-vscode/CHANGELOG.md]:
```suggestion
- Fix typo in doc title for [Lightning Web Components](forcedotcom.github.io/salesforcedx-vscode/articles/lwc/writing) ([PR #1830](https://github.com/forcedotcom/salesforcedx-vscode/pull/1830))
```

**Pull request merged**